demo/scss/examples.scss

```
@@ -21,3 +21,13 @@
   margin: var(--size-1) 0;
   padding: var(--size-5) 0;
 }
+
+.example-title {
+  margin-top: var(--size-2);
+  color: darkgray;
```

nit: I would use open-props variables for this color.

```suggestion
  color: var(--gray-6);
```